packages/lime-docs/files/lime-example

```diff
@@ -80,20 +80,20 @@ config lime wifi
 ### WiFi interface specific options ( override defaults options )
 
 ## use radio0 only for mesh
-config wifi radio0
+config wifi radio0 # check existing radios with "wifi status" command
```

Comment:
Shouldn't this be automatic?
Is this something that can be predicted?
If not, this setting can't come within the firmware, right?
Just curious on what would be the steps to use this functionality, and if we can make it simpler enough for the user to use (like adding wan connection on the lime-app for manual configs (wifi and manual ip and pppoe info)).

Reply:
and what about the usecase?
I never had to do this, so I don't see the case

Reply:
some usecases of specific radio configuration:

. use a radio as client
. use a radio just for mesh and another radio just for ap

Reply:
The comment is a bit misleading to me, i undestard it as if the option is meant to check for the existence of a radio, while it is not like that, I think the comment should explain what is the paramether for and then as an extra add "you should ensure the radio exists, for exaple usign `wifi status` command"

Reply:
@ilario those are not usecases, that is what you want to accomplish.
A usecase would be a case when a user needs that way of using the radio.
Like: As a user i want to get internet over wireless to share with the mesh
... Or something like that.
And with that, would be very interesting to know how frequent it is. Thanks!

Reply:
@G10h4ck improved

Reply:
@G10h4ck improved